index.js

```diff
@@ -15,7 +15,7 @@ const DEFAULT_CONFIG = {
   only: [],
   except: [],
   spacing: 2,
-  pretty: false,
+  format: "json",
```

Comment:
Keep the `pretty` option, and throw an error if both `pretty` and `format` options are set.  Same behavior as the interaction between `flat` and `rename`.

Reply:
addressed 👍🏼